src/panel_material_ui/chat/interface.py

```diff
@@ -78,11 +84,19 @@ def _click_send(
         instance: ChatInterface | None = None
     ) -> None:
         if self.disabled:
+            print("disabled")
+            return
+
+        if self.on_submit is not None:
+            print('submit')
+            self.on_submit(event, instance)
             return
-        objects = []
+
+        objects = self.active_widget.views
+        print(self.active_widget)
         if self.active_widget.value:
+            print(self.active_widget.value, "VALUE")  # why is this ""??
```

Comment:
@philippjfr I'm pretty stumped as to why:

When I run ChatInterface standalone, the widget's value properly gets processed...

https://github.com/user-attachments/assets/01be6801-694d-46c8-a855-88ce15e12b8f

However, when I run it with this branch (unified ChatAreaInput in welcome screen & chat interface) https://github.com/holoviz/lumen/pull/1372/files#diff-d99fd924b197130083f3cc0db28819f92a929aab5aef9f8d17a72b5a9aa7fbf5R281

https://github.com/user-attachments/assets/c382b7a7-90f5-46ca-807d-446254674a80

It only runs `print(self.active_widget)` and not `print(self.active_widget.value, "VALUE")` because `value == ""`